src/amo/containers/CategoryPage.js

```diff
@@ -14,10 +14,21 @@ export function CategoryPageBase(props) {
 }
 
 export function mapStateToProps(state, ownProps) {
+  // HACK: This loads Firefox personas (lightweight themes) for Android
```

Comment:
I would suggest changing this to `TODO` since we already have those. It would give us one less thing to grep for.